### ISSUE TITLE
wish to contact us? respond to this issue or read notes from the users.

### ISSUE THREAD

**Issue by LaughingSun** [nodeschool/bali#1]
Hello!

If you are interested in the next NodeSchool event, you have several options.
- Show your interest or help out with the  [next NodeSchool](https://github.com/nodeschool/bali/issues/2)
- If you don't have a GitHub account, you can susbcribe to the [NodeSchool  here. We will only send you information about upcoming events. Nothing more, we promise.
- You can watch this repository for new activity.

Best,
Erich

**Comment by RichardLitt** [comment]
We should add this information to the README instead of to this issue. What were you trying to link in point 2?

**Comment by krisnaw** [comment]
Hello 

I wanna join this event 

See you there

**Comment by RichardLitt** [comment]
Woo!

**Comment by maitrifischer** [comment]
I'd like to join too pls.

**Comment by RichardLitt** [comment]
Awesome! I'm going to update the website now, I've been really finding it hard to get time to do it. Give me a couple of minutes.

**Comment by RichardLitt** [comment]
@maitrifischer @krisnaw I added in an RSVP link! Email events@hubud and you'll be set to come. :)

**Comment by LaughingSun** [comment]
SO ready for the day?  I don't really see anything left to do until we get
there.  What time will you arrive? Also can you give me the name and number
of the HUBUD coordinator.

Talk to you soon.

On Fri, Apr 3, 2015 at 2:34 PM, Richard Littauer notifications@github.com
wrote:

> @maitrifischer https://github.com/maitrifischer @krisnaw
> https://github.com/krisnaw I added in an RSVP link! Email events@hubud
> and you'll be set to come. :)
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/nodeschool/bali/issues/1#issuecomment-89187001.

## 

Erich Horn
Denpasar, Bali, Indonesia
(62) 82140467070

**Comment by RichardLitt** [comment]
Pretty sure we're ready! I'll be there from around noon, probably, if not earlier - I basically am in Hubud all day, anyway.

**Comment by LaughingSun** [comment]
ok that was my plan was to arrive a little after 12 noon.  How will I find
you?

On Sat, Apr 4, 2015 at 8:26 AM, Richard Littauer notifications@github.com
wrote:

> Pretty sure we're ready! I'll be there from around noon, probably, if not
> earlier - I basically am in Hubud all day, anyway.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/nodeschool/bali/issues/1#issuecomment-89472539.

## 

Erich Horn
Denpasar, Bali, Indonesia
(62) 82140467070

**Comment by RichardLitt** [comment]
I'm the guy with the round glasses and short hair and beard, wearing a green GitHub shirt in Hubud. I'll be around. :)

**Comment by aportbury** [comment]
Just caught on to this group. Is there a schedule for the next meeting? I'm Canggu based and would love to join.

**Comment by RichardLitt** [comment]
Not at the moment. I'll be in the area again in early June, though, so maybe we could have another one? @LaughingSun what do you think?

**Comment by LaughingSun** [comment]
yes, we will have another one.  I cannot get on to planing until may, so
that means we will plan it for June.  But if you all have interest to meet
before then go ahead and arrange it.
Let me know how I can assist.

On Sun, Apr 19, 2015 at 4:35 PM, Richard Littauer notifications@github.com
wrote:

> Not at the moment. I'll be in the area again in early June, though, so
> maybe we could have another one? @LaughingSun
> https://github.com/LaughingSun what do you think?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/nodeschool/bali/issues/1#issuecomment-94251891.

## 

Erich Horn
Denpasar, Bali, Indonesia
(62) 82140467070

**Comment by RichardLitt** [comment]
Yeah. I won't be there in May, I think - I definitely will be in June. The 11th-13th might be a good time for one.

**Comment by LaughingSun** [comment]
sounds good. I will check out the other venues, maybe you can see about
hubud.

On Tue, Apr 21, 2015 at 11:38 AM, Richard Littauer <notifications@github.com

> wrote:
> 
> Yeah. I won't be there in May, I think - I definitely will be in June. The
> 11th-13th might be a good time for one.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/nodeschool/bali/issues/1#issuecomment-94628834.

## 

Erich Horn
Denpasar, Bali, Indonesia
(62) 82140467070

**Comment by aportbury** [comment]
Is everyone Ubud based? I know the Bali IT Meetup group has had a couple of meetings at Lineup Hub on Sunset Road.

**Comment by RichardLitt** [comment]
I'm not sure, at the moment. I was Ubud based - @LaughingSun is down in Denpasar.

**Comment by LaughingSun** [comment]
I am closer to Ubud than Sunset road or Sanur.  But we can definitely have
a meeting down there, How about in July?

On Mon, Apr 27, 2015 at 3:11 PM, Richard Littauer notifications@github.com
wrote:

> I'm not sure, at the moment. I was Ubud based - @LaughingSun
> https://github.com/LaughingSun is down in Denpasar.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/nodeschool/bali/issues/1#issuecomment-96526584.

## 

Erich Horn
Denpasar, Bali, Indonesia
(62) 82140467070

**Comment by aportbury** [comment]
July works for me. I'm flexible with location - It's nice to visit Ubud every now and then if that makes it easier for others.

**Comment by djchrispurple** [comment]
I'm interested to join.
cheers !

**Comment by LaughingSun** [comment]
Hi Richard,
It seems I am getting more and more onmy plate right now.  I was wondering
what your plans are for home base?  Stay in Bali or Thailand or NY? Anyway
maybe you can make some recommendation for someone (or yiourself) to temp
take over mentor title for the Bali group.  Anyone else have nominations?

Thanks for your support.

Erich

On Wed, Jun 3, 2015 at 2:16 PM, djchrispurple notifications@github.com
wrote:

> I'm interested to join.
> cheers !
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/nodeschool/bali/issues/1#issuecomment-108207019.

## 

Erich Horn
Denpasar, Bali, Indonesia
(62) 82140467070

**Comment by RichardLitt** [comment]
I'm actually about to move to Boston for a few months, and I'm not sure when I'll be back in Bali. So I'm not the best for the mentor title - although I think that's a bit of a misnomer. NodeSchool is autonomous; anyone who wants to step in and host an event is free to do so. I hope someone else does here in Bali, as we already have some resources set up to make that easier.

Good luck on the work @LaughingSun :)

**Comment by purwnt** [comment]
When events like this will be held in Jakarta?

**Comment by djchrispurple** [comment]
Hi guys
I could probably help.
I live in Bali and got 5/6 friends developers.
Cheers
Chris

**Comment by krisnaw** [comment]
@djchrispurple where do you stay? Ubud?

**Comment by emirnasution** [comment]
why bali, not jakarta ???
event in jakarta will be great !!!
please next event in Jakarta indonesia too ...

**Comment by RichardLitt** [comment]
@emirnasution You can check out the Jakarta branch here: https://github.com/nodeschool/jakarta

**Comment by balinux** [comment]
hi guys, can i join?
